File: src/backend/GetToKnowGame/Models/Question.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace GetToKnowGame.Models
{
    public class Question
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; } = string.Empty;

        [BsonElement("section")]
        public string Section { get; set; } = string.Empty;

        [BsonElement("questionText")]
        public string QuestionText { get; set; } = string.Empty;
    }
}

File: src/backend/get-to-know-game-go/models/question.go
package models

import "go.mongodb.org/mongo-driver/bson/primitive"

// Question represents a question in the game
type Question struct {
	ID           primitive.ObjectID `bson:"_id,omitempty" json:"id"`
	Section      string             `bson:"section" json:"section"`
	QuestionText string             `bson:"questionText" json:"questionText"`
}
